frontend/static/js/map.js

```diff
@@ -110,6 +110,43 @@ Osgende.BaseMapControl = function(settings) {
     });
   }
 
+  // extract relation IDs from feature
+  function get_relation_ids(feature) {
+    return feature.getProperties()['relations'];
+  }
+
+  function map_clicked(evt) {
+    // forEachFeatureAtPixel only works for visible features so we se a nearly transparent style and render once
+    obj.vroute_layer.setStyle( new ol.style.Style({
+                                    stroke: new ol.style.Stroke({
+                                            color: [200, 200, 200, 0.1],
+                                            width: 15,
+                                            }),
+                                     zindex: 1
+                                }));
+
+    obj.vroute_layer.once('render', function() { // wait for a callback
+      var relations = [];
+      obj.map.forEachFeatureAtPixel(evt.pixel, function onOpenDetails(feature, layer) {
+        var rels = get_relation_ids(feature);
+        if(rels)
+          relations = relations.concat(rels);
+      });
+      if(relations.length === 1)
```

Comment:
General style comment: please leave a space after 'if', 'for' etc and adjust bracket style to rest of code (opening bracket on end of line).